opus/application/static_media/js/opus.js

```diff
@@ -103,10 +103,9 @@ var opus = {
             return;
         }
 
-        // if (!$.isEmptyObject(opus.selections) || !opus.checkIfDrawnWidgetsAreDefault() || !opus.checkIfMetadataAreDefault()) {
-        if ($.isEmptyObject(selections) || !opus.checkIfDrawnWidgetsAreDefault()) {
+        if (!$.isEmptyObject(selections) || !o_utils.areObjectsEqual(opus.prefs.widgets, opus.default_widgets)) {
             $(".op-reset-button button").prop("disabled", false);
-        } else  if (!opus.checkIfMetadataAreDefault()) {
+        } else if (!o_utils.areObjectsEqual(opus.prefs.cols, default_columns.split(','))) {
```

Comment:
i think since you are checking for default metadata columns multiple times, keeping the function but updating may be better plan.  Same comment for drawnWidgets.

Reply:
Put back areDrawnWidgetsDefault and areSelectedMetadataDefault functions for readability.